demo-ui/app/uk/gov/ons/addressIndex/demoui/utils/RelativesExpander.scala

```diff
@@ -0,0 +1,53 @@
+package uk.gov.ons.addressIndex.demoui.utils
+
+import java.util.UUID
+
+import uk.gov.ons.addressIndex.model.db.index.{ExpandedRelative, ExpandedSibling, Relative}
+import uk.gov.ons.addressIndex.model.server.response.{AddressByUprnResponseContainer, AddressResponseRelative}
+import javax.inject.{Inject, Singleton}
+import uk.gov.ons.addressIndex.demoui.client.AddressIndexClientInstance
+import uk.gov.ons.addressIndex.model.AddressIndexUPRNRequest
+
+import scala.language.postfixOps
+import scala.concurrent.{Await, ExecutionContext, Future}
+import scala.concurrent.duration._
+import scala.util.{Failure, Success, Try}
+
+@Singleton
+class RelativesExpander @Inject ()(
+  apiClient: AddressIndexClientInstance
+)(implicit ec: ExecutionContext) {
+
+  def expandRelatives(apiKey: String, relatives: Seq[AddressResponseRelative]): Seq[ExpandedRelative] = {
+    relatives.map{rel => expandRelative(apiKey,rel)}
+  }
+
+  def expandRelative (apiKey: String, rel: AddressResponseRelative):  ExpandedRelative = {
+    ExpandedRelative (
+      rel.level,
+      getExpandedSiblings(apiKey, rel.siblings)
+    )
+  }
+
+  def getExpandedSiblings(apiKey: String, uprns: Seq[Long]): Seq[ExpandedSibling] = {
+    uprns.map(uprn => {
```

Comment:
new not required:
uprns.map(uprn => ExpandedSibling(uprn,Await.result(getAddressFromUprn(apiKey,uprn), 1 seconds)))